packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -157,6 +157,9 @@ export const getComponentCss = (
         // Necessary to make focus outlines fully visible
         padding: '4px',
         margin: '-4px',
+        // Fix scrollbar issues when slotted content includes .sr-only styles (see issue #3042)
+        transform: 'translate3d(0,0,0)',
+        zIndex: 1,
```

Comment:
Why is a `zIndex` needed to fix the overflow issue?

Reply:
Because otherwise the Accordion heading would render above overflowing content, see VRT:

<img width="537" alt="Bildschirmfoto 2024-03-07 um 13 27 24" src="https://github.com/porsche-design-system/porsche-design-system/assets/40788023/8a8a6f50-0ac5-4f48-8620-12306511a8f2">